modules/rosa-unlinking-and-deleting-ocm-and-user-iam-roles.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-deleting-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-unlinking-and-deleting-ocm-and-user-iam-roles_{context}"]
+= Unlinking and deleting the {cluster-manager} and user IAM roles
+
+If you installed a {product-title} (ROSA) cluster by using {cluster-manager-first}, you created {cluster-manager} and user Identity and Access Management (IAM) roles and linked them to your Red Hat organization. After deleting your cluster, you can unlink and delete the roles by using the ROSA CLI (`rosa`).
+
+[IMPORTANT]
+====
+The {cluster-manager} and user IAM roles are required if you want to install and manage other ROSA clusters in the same AWS account by using {cluster-manager}. You must only remove the roles if you no longer need to install ROSA clusters in your account by using {cluster-manager}.
```

Comment:
suggestion:
The {cluster-manager} and user IAM roles are required if you want to install and manage other ROSA clusters in the same AWS account. You must only remove the roles if you no longer need to install ROSA clusters in your AWS account.

Reply:
My understanding is that the OCM and user IAM roles are only required when using the OCM UI to install clusters. I think we need to explicitly state this in this note, because you can still install ROSA clusters through the CLI without those roles.

I will work on the wording to try to improve the flow of the note.

Reply:
I have updated the note so that it flows better. It is still specific to OCM-based deployment scenarios only:

> **Note**: The {cluster-manager} and user IAM roles are required if you want to use {cluster-manager} to install and manage other ROSA clusters in the same AWS account. You must only remove the roles if you no longer need to use {cluster-manager} to install ROSA clusters.